### ISSUE TITLE
How to use both 'reactitve' and 'computed' in 'useVuelidate'?

### ISSUE THREAD

**Issue by ddenev** [vuelidate/vuelidate#1105]
I am using the composition API:

```js
const data = reactive({
  type: null,
  name: '',
  currentBalance: null,
  creditCard: {
    targetMonth: new Date().getMonth(),
    targetYear: new Date().getFullYear()
  }
})
const v$ = useVuelidate({
  type: { required },
  name: { required },
  currentBalance: { required }
}, data)
```

Now I want to validate a computed ref which depends on both `data.creditCard.targetMonth` and `data.creditCard.targetYear`:
```js
const creditCardTargetDate = computed(() => {
  return new Date(data.creditCard.targetYear, data.creditCard.targetMonth + 1, 0)
})
```

How do I add the computed to the `useVuelidate` call above?

I don't want to 'pollute' my `data` with a pseudo-field just to satisfy the `useVuelidate` call.

I tried:
```js
const v$ = useVuelidate({
  type: { required },
  name: { required },
  currentBalance: { required }
}, { data, creditCardTargetDate })
```
but it did not work.

**Comment by dobromir-hristov** [comment]
You could call useVuelidate another time for your computed stuff only, or as you have shown add it to the state. In that case however, the validations object has to match the state format.

**Comment by ddenev** [comment]
Thank you, @dobromir-hristov , that was the other approach that I also was thinking of but it didn't feel quite intuitive to me.

Do you plan to extend vuelidate to allow mixed types of state in the same `useVuelidate` call? This way we could keep the validations setup in a single tidy object rather than have separate ones.

**Comment by dobromir-hristov** [comment]
Did you try this?
```js
const v$ = useVuelidate({
 data: { 
   type: { required },
   name: { required },
   currentBalance: { required }
 },
 computeds: {
   creditCardTargetDate: { fooValidator }
 }
}, 
{ 
  data, 
  computeds: {
    creditCardTargetDate 
  }
})
```

**Comment by ddenev** [comment]
That actually works! Thank you, @dobromir-hristov.

I think it would be very useful to add this example to the docs - https://vuelidate-next.netlify.app/advanced_usage.html#composition-api

**Comment by dobromir-hristov** [comment]
Contributions are welcome. I haven't had much time for the library lately.

**Comment by dobromir-hristov** [comment]
fixed by #1106
